eng/common/pipelines/templates/steps/verify-readmes.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+- name: PackagePropertiesFolder
+  type: string
+- name: RepoRoot
+  type: string
+  default: $(Build.SourcesDirectory)
+- name: SettingsPath
+  type: string
+  default: '$(Build.SourcesDirectory)/eng/.docsettings.yml'
+- name: DocWardenVersion
+  type: string
+  default: ''
+- name: Condition
+  type: string
+  default: succeeded()
+
+steps:
+- pwsh: |
+    $packageProperties = Get-ChildItem -Recurse -Force "${{ parameters.PackagePropertiesFolder }}" `
+      | Where-Object { $_.Extension -eq '.json' } `
```

Comment:
Are there other files? Should we just do `Get-ChildItem -Recurse "${{ parameters.PackagePropertiesFolder }}" *.json`?

Reply:
I would normally say no. but my thought was to at least do some basic protection against artifacts getting dropped in and around the folder. "artifacts" being stuff from 1ES runs, etc.